### PR TITLE
vishakha/Axpy pipeline with variable

### PR DESCRIPTION
**Pull request by vishakha-nervana**

### CODE REVIEW

File: test/python/test_axpy_var_pipelined.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#sys.path.insert(0, '../../examples')
+
+from axpy_var_pipelined import *
+
+
+class TestAxpyPipelined(NgraphTest):
+
+    def test_axpy_pipelined(self):
+        #prefetch_env = "NGRAPH_TF_USE_PREFETCH"
+        ngraph_backend_i = "NGRAPH_TF_BACKEND"
```
Comment (shresthamalik) [inline test/python/test_axpy_var_pipelined.py]: dont need to set backend now as cpu will run through parallel exec

File: examples/axpy_var_pipelined.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    var_sum = 11
+    with tf.Session() as sess:
+        # Initialize the globals and the dataset
+        sess.run(tf.global_variables_initializer())
+        sess.run(iterator.initializer)
+
+        for i in range(1, 10):
+            # Expected value is:
+            expected_output_array[i - 1] = (
+                (input_array[i - 1] * multiplier) * 5) + var_sum
```
Comment (shresthamalik) [inline examples/axpy_var_pipelined.py]: as discussed the variable does not seem to be updated every iteration

File: test/python/test_axpy_var_pipelined.py (hunk 1 above)
Comment (vishakha-nervana) [reply test/python/test_axpy_var_pipelined.py]: Removed this env

File: examples/axpy_var_pipelined.py (hunk 2 above)
Comment (vishakha-nervana) [reply examples/axpy_var_pipelined.py]: added assign to ensure variable update per iteration.

File: examples/axpy_pipelined.py [hunk 3]
```diff
@@ -61,7 +61,6 @@ def run_axpy_pipeline():
     pipeline, iterator = build_data_pipeline(input_array, map_function,
                                              batch_size)
     model = build_simple_model(pipeline)
-
     with tf.Session() as sess:
         # Initialize the globals and the dataset
         sess.run(tf.global_variables_initializer())
```
Comment (shresthamalik) [inline examples/axpy_pipelined.py]: if this script doesnt have variables we dont need this line . I removed it in my PR
sess.run(tf.global_variables_initializer())

File: examples/axpy_var_pipelined.py [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
```
Comment (shresthamalik) [inline examples/axpy_var_pipelined.py]: [minor]5 and 1 can be made constants and passed to the model.

File: test/python/test_axpy_var_pipelined.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+#sys.path.insert(0, '../../examples')
+
+from axpy_var_pipelined import *
+
+
+class TestAxpyPipelined(NgraphTest):
+
+    def test_axpy_pipelined(self):
+        #prefetch_env = "NGRAPH_TF_USE_PREFETCH"
+        #env_var_map = self.store_env_variables([prefetch_env, ngraph_backend_i])
```
Comment (shresthamalik) [inline test/python/test_axpy_var_pipelined.py]: stray hanging ngraph_backend_i

Comment (vishakha-nervana) [reply test/python/test_axpy_var_pipelined.py]: will get rid of it

File: examples/axpy_pipelined.py (hunk 3 above)
Comment (vishakha-nervana) [reply examples/axpy_pipelined.py]: removed

File: examples/axpy_var_pipelined.py (hunk 4 above)
Comment (vishakha-nervana) [reply examples/axpy_var_pipelined.py]: done.

File: test/python/test_axpy_var_pipelined.py (hunk 5 above)
Comment (vishakha-nervana) [reply test/python/test_axpy_var_pipelined.py]: done.

File: examples/axpy_pipelined.py (hunk 3 above)
Comment (shresthamalik) [reply examples/axpy_pipelined.py]: please remove the setting of backend interpreter in this script also

File: examples/axpy_var_pipelined.py [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+def main(_):
+    input_array, output_array, expected_output_array = run_axpy_pipeline()
+    for i in range(1, 10):
+        print("Iteration:", i, " Input: ", input_array[i - 1], " Output: ",
+              output_array[i - 1], " Expected: ", expected_output_array[i - 1])
+        sys.stdout.flush()
+
+
+if __name__ == '__main__':
+    os.environ['NGRAPH_TF_BACKEND'] = "INTERPRETER"
```
Comment (shresthamalik) [inline examples/axpy_var_pipelined.py]: This needs to be removed. The script can run on CPU

**Pull request merged**